modules/gateways/blockonomics.php

```diff
@@ -84,11 +83,43 @@ function reqListener () {
 				newBtn.disabled = false;
 			}
 
+			function reqListener () {
+				responseObj = {};
+				switch (this.status) {
+					case 200:
+						responseObj = JSON.parse(this.responseText);
+						break;
+					case 401:
```

Comment:
Lets have only one default,  and one message. 
Unable to locate/execute *test_setup_url*. Contact blockonomics support for help